Remora.Discord.Commands/Extensions/CommandTreeExtensions.cs

```diff
@@ -239,12 +265,24 @@ public static Result<IReadOnlyList<IBulkApplicationCommandData>> CreateApplicati
                     commandType = commandNode.GetCommandType();
 
                     // Top-level command outside of a group
-                    var permissionAttribute = commandNode.GroupType
-                        .GetCustomAttribute<DiscordDefaultPermissionAttribute>();
+                    var memberPermissionsAttribute = commandNode.GroupType
```

Comment:
Weird formatting here. Don't add line breaks unless you need to, and prefer breaking before operators like `??`.